### ISSUE TITLE
problem of make file upload with remove link 

### ISSUE THREAD

**Issue by wanyunli** [getgrav/grav-plugin-form#201]
Hi, I am creating a form with file upload field. In the file upload field I would like to have the remove link. I checked the form plugin code, currently in the form.min.js file, the addRemoveLinks is false. How could I config to make the addRemoveLinks config to be true? 

When I debugging I manually set the addRemoveLinks to be true in form.min.js file, and when I remove a uploaded file, in the console shows a error that something similar "c function is not defined." , which is responsible for remove the temp uploaded file( if I understand right ). Could that be fixed also ?

Thank you very much !

**Comment by twsowerby** [comment]
I would also like to know how to do this, so hopefully this bumps the questions.

I've got a front end form with a file upload field. I've set it to only allow one file to be uploaded. What I'm trying to do is allow the user to delete the one file from the upload field and upload a different single file (use case: they uploaded wrong file by accident and need to replace it with the correct one)

I've worked with Dropzone in the past and I know it has the ability to remove files, I just can't quite work out how to do it with the way the Grav Forms plugin has implemented it.

Thanks,

Tom
